sections/semantics-embedded-content.include

```diff
@@ -6124,8 +6124,8 @@ zero or more <{track}> elements, then
   var support = videoElement.canPlayType('video/x-new-fictional-format;codecs="kittens,bunnies"');
   if (support != "probably") {
     // not confident of browser support
-    // but we have a plugin
-    // so use plugin instead
+    // but we might have a plugin
+    // so try plugin instead
```

Comment:
If we are going to discourage probing for content type support in plug-ins, maybe we shouldn't promote the usage at all? This effectively be the same as toss to plug-in and cross fingers real hard. Not sure if the spec example should promote this.